### PR TITLE
Check in/54719/pci conf accordion

### PR DESCRIPTION
**Pull request by acrollet**
## Summary

- Updates application to not show the "how to update your information" accordion item when all confirmation pages are skipped because demographics are already up to date

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#54719

## Testing done

- Manual testing, added e2e test 

## What areas of the site does it impact?
- pre-check-in complete page

## QA Instructions

To test, go to pre-check-in with the new uuid for all confirmations up to date: `e544c217-6fe8-44c5-915f-6c3d9908a678`

- [ ] the accordion should not be present on the /complete page.
- [ ] When a Veteran answers yes to all questions, this accordion should not be present on the /complete page.

go to pre-check-in with the new uuid for only demographic confirmations up to date: `7397abc0-fb4d-4238-a3e2-32b0e47a1527`

- Answer "no" for emergency contact and next of kin
- [ ] You should see the "how to update" information for EC and NoK, but not demographics

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature
- [ ]  [Accessibility foundational testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/check-in/components/PreCheckInAccordionBlock.jsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ const PreCheckInAccordionBlock = ({
         <strong>{title}</strong>
         {appointmentType === 'clinic' ? (
           <p>
-            t('a-staff-member-will-help-you-on-the-day-of-your-appointment')
```
Comment (brianseek) [inline src/applications/check-in/components/PreCheckInAccordionBlock.jsx:99]: nice catch! was this just rendering out this literal text before?

File: src/applications/check-in/pre-check-in/tests/e2e/confirmation-display/confirmation.display.cypress.spec.js [hunk 2]
```diff
@@ -47,7 +51,7 @@ describe('Pre-Check In Experience', () => {
       Confirmation.validatePageContent();
       cy.injectAxeThenAxeCheck();
     });
-    it('How can I update me information accordion is not visible', () => {
```
Comment (brianseek) [inline src/applications/check-in/pre-check-in/tests/e2e/confirmation-display/confirmation.display.cypress.spec.js:50]: argh 🏴‍☠️

**Review by brianseek — APPROVED** [review]
Looks good!

| Before screenshot: | After screenshot: |
| ---- | ---- |
| localhost_3001_health-care_appointment-pre-check-in_complete (1) | localhost_3001_health-care_appointment-pre-check-in_complete (2) |

File: src/applications/check-in/components/PreCheckInAccordionBlock.jsx (hunk 1 above)
Comment (acrollet) [reply src/applications/check-in/components/PreCheckInAccordionBlock.jsx:99]: yup, @benbrasso-agile6 found it!

File: src/applications/check-in/pre-check-in/tests/e2e/confirmation-display/confirmation.display.cypress.spec.js (hunk 2 above)
Comment (acrollet) [reply src/applications/check-in/pre-check-in/tests/e2e/confirmation-display/confirmation.display.cypress.spec.js:50]: 🦜

**Comment by loripusey** [comment]
@acrollet  the screenshot above shows the problem with the display of this content; was that fixed?
image

**Comment by acrollet** [comment]
@loripusey shoot, good question - I had fixed it, not sure how Brian saw it - I will confirm on staging today.

**Comment by brianseek** [comment]
@loripusey / @acrollet the first screenshot was from master at the time. This PR fixed it.

**Pull request merged**